auth/v1.js

```diff
@@ -134,14 +128,14 @@ api.declare({
 }, async function(req, res) {
   let clientId = req.params.clientId;
 
-  // Load client
-  let client = await this.Client.load({clientId}, true);
+  // Load client from the resolver's cache
+  let client = this.resolver.jsonClient(clientId);
```

Comment:
I have mixed feeling about loading the clients from memory.. notably, they could be slightly out of date as it takes a few ms for the pulse message purging the client to arrive... And a few ms to reload it...

So we now have race conditions in the editor that could result in someone seeing and old client.. IMO the purge of clients from memory based on pulse message is sort of a best effort service..

There is a reason we purge all clients every 20 minutes, that because then we don't have to worry about loosing a push message. In this case it would definitely feel safer if we loaded from azure...

It's not like the cost of loading for this purpose is significant. The cost of having out-of-date information is however a lot higher... Out of date authentication by 20min is not ideal, but certainly more acceptable.